src/main/java/com/hbm/world/WorldUtil.java

```diff
@@ -31,6 +34,38 @@ public static void syncBiomeChange(World world, Chunk chunk) {
 		PacketDispatcher.wrapper.sendToAllAround(new BiomeSyncPacket(coord.chunkXPos, coord.chunkZPos, chunk.getBiomeArray()), new TargetPoint(world.provider.dimensionId, coord.getCenterXPos(), 128, coord.getCenterZPosition() /* who named you? */, 1024D));
 	}
 
+    /**Chunkloads the chunk the entity is going to spawn in and then spawns it
+	 * @param entity The entity to be spawned**/
+
+	/*fun fact: this is based off of joinEntityInSurroundings in World
+	  however, since mojang is staffed by field mice, that function is client side only and half-baked
+	 */
+	public static void loadAndSpawnEntityInWorld(Entity entity) {
```

Comment:
If this code was clientside only, are you sure replicating it is going to work on the server? Did you check if any of the called functions are also clientside only or behave different on local worlds, and did you test it on a server?

Reply:
I checked all the code, there was literally no reason whatsoever for any of it to be on client-side, all it did was get a 2x2 chunk area, so it'd load, and then spawn the entity (which it didn't even do properly), I also have a server (thanks pecora's SMP) running my fork, and so far they've had no issues with the airstrikes.